SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -23,7 +23,21 @@ DEFINE_LOG_CATEGORY(LogSpatialDeploymentManager);
 
 #define LOCTEXT_NAMESPACE "FLocalDeploymentManager"
 
-static const FString SpatialServiceVersion(TEXT("20190930.180414.3b04a59226"));
+static const FString SpatialExe(TEXT("spatial.exe"));
```

Comment:
Why is this here? Methods should be using `FSpatialGDKServicesModule::GetSpatialExe()` in 0.7

Reply:
Good spot, will remove